src/v/cluster/partition_balancer_backend.cc

```diff
@@ -62,18 +62,14 @@ partition_balancer_backend::partition_balancer_backend(
   , _timer([this] { tick(); }) {}
 
 void partition_balancer_backend::start() {
-    if (is_enabled()) {
```

Comment:
does this need a feature barrier? Like in a mixed mode cluster during upgrade, the controller leader is temporarily on an upgraded node and then back to a node with old bits.. wondering if that ping pong can cause any weird issues.. don't think it does but just wanted to double check.

Reply:
This is probably harmless - worst case we'll schedule decom movements both from members_backend and the partition balancer